projects/bitcoin-core/Dockerfile

```diff
@@ -14,7 +14,11 @@
 #
 ################################################################################
 
-FROM gcr.io/oss-fuzz-base/base-builder
+# Using Ubuntu 16.04 because of breakage on Ubuntu 20.04.
+# See https://github.com/google/oss-fuzz/issues/6291 for more details.
+FROM gcr.io/oss-fuzz-base/base-builder:xenial
```

Comment:
Is the focal image already available, so that we can start debugging this?

Reply:
It's not fully ready yet. We'll ping you once it is. Thanks!